### PR TITLE
BNH-135&121 feat: standardise links in tables

### PR DESCRIPTION
**Pull request by cec200wire**

### CODE REVIEW

File: web/Views/Admin/PropertyList.cshtml [hunk 1]
```diff
@@ -53,13 +53,20 @@
             }
             
             <td>
-                <a type="button" class="btn btn-outline-primary" value="@property.PropertyId" href="@Url.Action("ViewProperty", "Property", new { id = property.PropertyId })">
-                    <i class="bi bi-house"></i>
+                <a type="button" class="btn btn-outline-primary" value="@property.LandlordId" href="@Url.Action("Profile", "Landlord", new { id = property.LandlordId })">
```
Comment (oskarwilliams378) [inline web/Views/Admin/PropertyList.cshtml:56]: Do we need this value? And do we need it in the other places we are also setting this value?

**Review by oskarwilliams378 — CHANGES_REQUESTED** [review]
Just a minor comment :)

File: web/Views/Admin/PropertyList.cshtml (hunk 1 above)
Comment (cec200wire) [reply web/Views/Admin/PropertyList.cshtml]: We don't need that value - the routing is conveying all the information now, so setting the value with value= is unnecessary. I've removed it (here, and in a couple of other places).

Comment (oskarwilliams378) [reply web/Views/Admin/PropertyList.cshtml]: Thanks :)

**Review by oskarwilliams378 — APPROVED** [review]

**Pull request merged**